manifests/init.pp

```diff
@@ -1,3 +1,24 @@
+# == Class: couchdb
+#
+# Install and manage the CouchDB server
+#
+# === Parameters
+#
+# None
+#
+# === Examples
+#
+#  include ::couchdb
+#
+# === Authors
+#
+# CampToCamp SA <http://www.camptocamp.com/>
+#
+# === Copyright
+#
+# Copyright 2015 Bren Briggs, unless otherwise noted.
```

Comment:
Well... not sure what happened there. Doesn't seem to matter anyway, so I'll just forget about this PR.